qutebrowser/config/configtypes.py

```diff
@@ -806,19 +819,17 @@ def validate(self, value):
                 raise configexc.ValidationError(value, "may not be empty!")
         value = os.path.expanduser(value)
         try:
+            if not os.path.isabs(value):
+                cfgdir = standarddir.config()
+                if cfgdir and os.path.isfile(os.path.join(cfgdir, value)):
+                    return
+                raise configexc.ValidationError(value,
+                                                "must be an absolute path!")
```

Comment:
I think the error message(s) should be a bit more fine-grained here, otherwise it might be confusing.

I'd say:

- `if cfgdir is None:`, raise a `ValidationError` with something like "must be an absolute path if no config directory is used!"
- If it's not a file, raise a `ValidationError` with something like "must be an existing file relative to the config directory"